src/jobsautoreport/main.py

```diff
@@ -27,35 +29,30 @@ def main() -> None:
     querier = Querier(client, index_name)
     reporter = Reporter(querier)
 
+    data = {}
+
     success_rate = reporter.get_success_rate(a_week_ago, now)
     number_of_jobs_triggered = reporter.get_number_of_jobs_triggered(a_week_ago, now)
     average_jobs_duration = reporter.get_average_duration_of_jobs(a_week_ago, now)
 
     if average_jobs_duration is None:
         return
 
-    slack_webhook_url = config.SLACK_WEBHOOK_URL
-    webhook = WebhookClient(slack_webhook_url)
-    res = webhook.send(
-        blocks=[
-            {
-                "type": "section",
-                "text": {"type": "mrkdwn", "text": "*Jobs Weekly Report:*"},
-            },
-            {
-                "type": "section",
-                "text": {
-                    "type": "mrkdwn",
-                    "text": (
-                        f"• Jobs success rate: _{success_rate}_\n"
-                        f"• Number of jobs triggered: _{number_of_jobs_triggered}_\n"
-                        f"• Jobs average duration: _{int(average_jobs_duration.total_seconds()) // 60}_ minutes, and _{int(average_jobs_duration.total_seconds())  % 60}_ seconds"
-                    ),
-                },
-            },
-        ]
+    data["success_rate"] = success_rate
+    data["number_of_jobs_triggered"] = str(number_of_jobs_triggered)
+    data["average_jobs_duration"] = str(average_jobs_duration)
+
+    slack_bot_token = config.SLACK_BOT_TOKEN
+    web_client = WebClient(token=slack_bot_token)
+
+    slack_message = SlackMessage(
+        channel_id=config.SLACK_CHANNEL_ID,
+        data=data,
     )
```

Comment:
this part should be in the `SlackMessenger` class, it'll be responsible the build and send a message from the data